src/targeting.js

```diff
@@ -260,7 +264,7 @@ export function newTargeting(auctionManager) {
             winner.sendStandardTargeting ||
             standardKeys.indexOf(key) === -1)
           .map(key => ({
-            [(key === 'hb_deal') ? `${key}_${winner.bidderCode}`.substring(0, MAX_DFP_KEYLENGTH) : key.substring(0, MAX_DFP_KEYLENGTH)]: [winner.adserverTargeting[key]]
+            [(key === CONSTANTS.TARGETING_KEYS.DEAL) ? `${key}_${winner.bidderCode}`.substring(0, MAX_DFP_KEYLENGTH) : key.substring(0, MAX_DFP_KEYLENGTH)]: [winner.adserverTargeting[key]]
```

Comment:
don't need the `CONSTANTS.`

Reply:
TARGETING_KEYS is an array, does not have a DEAL key.
I think we need to keep this code as it is.